Sources/ScipioKit/Producer/Compiler.swift

```diff
@@ -104,3 +112,9 @@ extension Package {
         buildArtifactsDirectoryPath(buildConfiguration: buildConfiguration, sdk: sdk).appendingPathComponent("\(target).framework.dSYM")
     }
 }
+
+extension ResolvedTarget {
```

Comment:
```suggestion
private extension ResolvedTarget {
```

If we don't use this other place, it is better to have it

Reply:
I'll use `fileprivate` instead